uber/models/admin.py

```diff
@@ -108,6 +114,30 @@ def is_expired(self):
         return self.generated < datetime.now(UTC) - timedelta(days=7)
 
 
+class AccessGroup(MagModel):
+    """
+    Sets of accesses to grant to admin accounts.
+    """
+    _NONE = 0
+    _LIMITED = 1
+    _CONTACT = 2
+    _FULL = 5
+    _READ_LEVEL_OPTS = [
+        (_NONE, 'Same as Read-Write Access'),
+        (_LIMITED, 'Limited'),
+        (_CONTACT, 'Contact Info'),
+        (_FULL, 'All Info')]
+    _WRITE_LEVEL_OPTS = [
+        (_NONE, 'No Access'),
+        (_LIMITED, 'Limited'),
+        (_CONTACT, 'Contact Info'),
+        (_FULL, 'All Info')]
+
+    name = Column(UnicodeText)
+    access = Column(JSONB, default={})
+    read_only_access = Column(JSONB, default={})
```

Comment:
JSONB allows us to edit the columns as if they were dictionaries. Hopefully there aren't any reasons to not just use it, since it made this change way easier to implement.